vermouth/parser_utils.py

```diff
@@ -34,8 +34,8 @@ class SectionParser(type):
     """
     def __new__(mcs, name, bases, attrs, **kwargs):
         obj = super().__new__(mcs, name, bases, attrs, **kwargs)
-        if not hasattr(obj, 'METH_DICT'):
-            obj.METH_DICT = {}
+        #if not hasattr(obj, 'METH_DICT'):
+        obj.METH_DICT = {}
```

Comment:
As described in the corresponding issue, I'm reasonably sure this is only a partial fix. Leave this as-is, but leave the issue open.